src/adapters/WstEthChainlinkAdapter.sol

```diff
@@ -0,0 +1,35 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity 0.8.21;
+
+import {IStEth} from "../interfaces/IStEth.sol";
+import {AggregatorV3Interface} from "../interfaces/AggregatorV3Interface.sol";
+
+import {ErrorsLib} from "../libraries/ErrorsLib.sol";
+
+/// @title WstEthChainlinkAdapter
+/// @author Morpho Labs
+/// @custom:contact security@morpho.org
+/// @notice wstETH/ETH exchange rate price feed.
+/// @dev This contract should only be used as price feed for `ChainlinkOracle`.
+contract WstEthChainlinkAdapter is AggregatorV3Interface {
+    uint8 public constant decimals = uint8(18);
+    string public constant description = "wstETH/ETH exchange rate";
+    uint256 public constant version = 1;
+
+    IStEth public immutable ST_ETH;
+
+    constructor(address stEth) {
+        require(stEth != address(0), ErrorsLib.ZERO_ADDRESS);
+        ST_ETH = IStEth(stEth);
+    }
+
+    function getRoundData(uint80) external view returns (uint80, int256, uint256, uint256, uint80) {
+        return latestRoundData();
+    }
```

Comment:
It feels weird to include this when it is specific to having the round id as input. Do we see a usecase for it ? (alternative is to revert because we do not use it in the oracle)

Reply:
I wonder the same question. I'm not sure about what's best tbh

Reply:
@morpho-org/onchain-reviewers thoughts?

Reply:
It seems traditional aggregators expose [empty data when an inexistent round is queried via this getter](https://github.com/smartcontractkit/libocr/blob/6359502d2ff1165c7e6b77b9eff2c5a46a7a4fbb/contract/OffchainAggregator.sol#L818-L849)

In our case, whether we revert or the current version of the code is fine to me